### ISSUE TITLE
Inspect unlinked html found in templates folder and unused/replaced files

### ISSUE THREAD

**Issue by carlosparadis** [erdl/survey_admin#3]
* _formhelpers.html
 * Admin_dashboard.html
 * Devicepage.html
 * Error_deployment.html
 * Error_page.html
 * Manage_questions.html
 * survery_mode.html

Decide what to do here once the purpose for these files is clear.

**Comment by jygh98** [comment]
# formhelpers.html
After further inspection of the html files, it appears that formhelpers acts as a template for question_form.html, survey_form.html, and deployment_form.html.  From what it appears like it was done like this to keep the files modular and we would actually need to keep this file.

**Comment by jygh98** [comment]
# admin_dashboard.html

When trying to access this web page, a build error occur 

    werkzeug.routing.BuildError
    werkzeug.routing.BuildError: Could not build url for endpoint 'admin_landing'. Did you mean 'question_page' instead?

With a further inspection of the html file, I noticed this section 

    <div class="navbar-header">
		<a class="navbar-brand" href="{{ url_for('admin_landing') }}">Home</a>

you would realize we do not have a routing url 'admin_landing', but what we do have is home.  So if you change the url_for to any existing routes, say home

     <div class="navbar-header">
		<a class="navbar-brand" href="{{ url_for('home') }}">Home</a>
This is the web page that appears

<img width="779" alt="admin_dashboard" src="https://user-images.githubusercontent.com/25787662/28696844-e0e9dfa8-72d3-11e7-8b71-4a4f92b9e17e.png">

So it looks like this was a feature to directly add questions through the dashboard, however if I were to create a test question it would lead me to another build error

    werkzeug.routing.BuildError: Could not build url for endpoint 'question_page'. Did you forget to specify values ['questionid']?

If I add some jinja logic to make use of the questionid to look like this

	<div class="navbar-header">
		{% for question in questions %}
		<a class="navbar-brand" href="{{ url_for('question_page', questionid=question.question_id) }}">Home</a>
	        {% endfor %}
	</div>

Also it looks like admin_dashboard makes use of the questionform, I am not completely sure how it works but if I were to link the action to /questionform 

    <form id="questionForm" name="questionForm" action="/questionform" method="POST" class="form-horizontal" onSubmit="validateForm()">

It will actually create questions, although i still have to figure out why it would not take my text inputs.  But it would create empty questions so that is interesting to note.
<img width="718" alt="sample" src="https://user-images.githubusercontent.com/25787662/28697412-6b189770-72d7-11e7-9ebe-6b74b45e65e2.png">

**Comment by jygh98** [comment]
# error_deployment.html
This is a functional webpath, it does seem that Allie commented out exception paths that would return this page such as in deployment_form():

    @app.route('/deploymentform', methods=['GET', 'POST'])
    @login_required
    def deployment_form():
      
    '''
        else:
            print(form.errors)
        #except:
        return render_template('error_deployment.html')
     '''

Not too sure why she decided to do this as currently on deployment any error page would lead to an 500 Internal Server Error.  Below is how the web page looks like.

<img width="728" alt="dsa" src="https://user-images.githubusercontent.com/25787662/28697844-a86525dc-72da-11e7-9fe2-065e51ca5171.png">

**Comment by jygh98** [comment]
# error_page

Again, same as before error_page appears to be commented out in the main code.  

<img width="860" alt="error" src="https://user-images.githubusercontent.com/25787662/28697910-2d996286-72db-11e7-8b41-54a5603c81ab.png">

**Comment by jygh98** [comment]
# manage_questions

Appears to be a functional webpath, however the views.py has to be modified slightly to pass the question objects to actually display the questions.  I am not too sure what is the point of this page as we do have /showquestions that accomplishes a similar purpose.

<img width="562" alt="manage" src="https://user-images.githubusercontent.com/25787662/28698051-55ba7876-72dc-11e7-8079-eb8b18231b36.png">

**Comment by jygh98** [comment]
# survey_mode
Looks like a test page of the survey mode that we have now.  Clicking the options does not do anything.  Looks like another test page.

<img width="961" alt="test" src="https://user-images.githubusercontent.com/25787662/28698142-023b57dc-72dd-11e7-80fe-2bce4950247d.png">

**Comment by jygh98** [comment]
# devicepage

Taking a look at devicepage.html, it looks like it makes use of the activequestionurl
    
     <div class="container">
		<h1>Device</h1>
		<p>{{ question.activequestionurl }}</p>
		<h1>Question URL</h1>
		<p>\{{ question.activequestionurl }}</p>

    <div class="form-group">
			{{ render_field(form.activequestionurl) }}

Which is part of the ActiveQuestionForm, and that uses of the ActiveQuestion object


    class ActiveQuestionForm(FlaskForm):
    urls=ActiveQuestion.query.with_entities(ActiveQuestion.activequestionurl)
    print("Hi")
    print(urls)
    activequestionurl = SelectField(u"activequestionurl", choices=[(q.activequestionid, q.activequestionurl) for q in ActiveQuestion.query.order_by('activequestionurl')])

From devicepage.html, it looks like its purpose is to display any active questions that are currently deployed.

**Comment by jygh98** [comment]
@carlosparadis 
Upon revisiting this issue it seems that we can separate the various files as such

Keep
- formhelpers.html

Remove
- admin_dashboard.html - this was a previous feature that would allow us to directly add questions from surveyadmin, however due to security reasons we decided it was better to add questions through the database instead
- errordeployment.html and errorpage.html - both are currently commented out  and not in use
- manage_questions.html - webpath is not in used and appears to be now replaced by /showquestions
- survey_mode.html - interesting webpath as it looks like we were trying to conduct surveys on surveyadmin and is simply a test page.  Not in use
- devicepage.html - functionality is not in use, appears to been able to display active questions that were deployed (ActiveQuestionForm has already been commented out)

**Comment by carlosparadis** [comment]
blocked by #23, we can only add questions after we implement a safer authentication protocol.

**Comment by carlosparadis** [comment]
@ryantanaka rotating this to you. Keeping as low priority, and more as final polishing once other higher importance issues are done.
